src/cf-tables.less

```diff
@@ -0,0 +1,484 @@
+// Import external dependencies
+
+@import '../../cf-core/src/cf-core.less';
+
+/* ==========================================================================
+   Capital Framework
+   Table Styling
+   ========================================================================== */
+
+
+
+/* topdoc
+  name: Tables
+  family: cf-tables
+  patterns:
+    - name: Table base style (inherited from cf-core)
+      markup: |
+        <table>
+            <thead>
+                <tr>
+                    <th>Column 1 header</th>
+                    <th>Column 2 header</th>
+                    <th>Column 3 header</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>Row 1, column 1</td>
+                    <td>Row 1, column 2</td>
+                    <td class="table__right-align">13</td>
+                </tr>
+                <tr>
+                    <td>Row 2, column 1</td>
+                    <td>Row 2, column 2</td>
+                    <td class="table__right-align">23</td>
+                </tr>
+                <tr>
+                    <td>Row 3, column 1</td>
+                    <td>Row 3, column 2</td>
+                    <td class="table__right-align">33</td>
+                </tr>
+            </tbody>
+        </table>
+      notes:
+        - "The table is not responsive without adding one of the 'small screen'
+            classes below."
```

Comment:
That makes sense. I think I might just do away with the base styling example. Or give more emphasis to the fact that it's not responsive and we STRONGLY recommend using one of the responsive classes.